pkgs/servers/web-apps/codimd/js-sequence-diagrams/default.nix

```diff
@@ -11,9 +11,11 @@ in
     name = "js-sequence-diagrams";
     packageName = "js-sequence-diagrams";
     version = "1000000.0.6";
-    src = pkgs.fetchurl {
-      url = "https://registry.npmjs.org/js-sequence-diagrams/-/js-sequence-diagrams-1000000.0.6.tgz";
-      sha1 = "e95db01420479c5ccbc12046af1da42fde649e5c";
+    src = pkgs.fetchFromGitHub {
+      owner = "Moeditor";
```

Comment:
IIUC we can't use upstream directly because [package.json](https://github.com/bramp/js-sequence-diagrams/blob/5cd8f76d0861f0845599ab65dd856e5dbff73e47/package.json) is missing is the field "version", so instead of picking a random fork, what do you of think forking bramp/js-sequence-diagrams under your username and adding a arbitrary `version`?

Reply:
This isn't a random fork, this is basically the package we've used all the time, but now it's the git source rather than the (unpublished) NPM tarball.

I'm a bit hesitant to use `bramp/js-sequence-diagrams`, except for the package.json issue I'm not sure how much effort it'll take to get a working NPM build that is usable by `codimd`'s build process.